### DISCUSSION TITLE
Handle resolution for *.bsky.social handles

### DISCUSSION THREAD

**Discussion by snarfed** [bluesky-social/atproto#1661] [How to do it?] [answered]
Hi again! `*.bsky.social` handles, the default handles you get when you sign up, don't serve [handle resolution](https://atproto.com/specs/handle#handle-resolution) via either the DNS or HTTPS methods. OK. For those of us building tools that do handle resolution from scratch, do you have any guidance on how we should handle `*.bsky.social` handles? Should we special case them for now? Or maybe in general, if a handle fails resolution and it's a subdomain, see if its parent domain serves `com.atproto.server.describeServer` and returns the parent domain in `availableUserDomains`, and if so, call `com.atproto.identity.resolveHandle` there? Or...?

**Comment by DavidBuchanan314** [comment, marked as answer]
I don't think that's true, they appear to serve via the standard HTTPS mechanism. For example: https://abcdef.bsky.social/.well-known/atproto-did

**Reply by snarfed** [reply]
Huh, you're right. I wonder what I did wrong when I originally tested this. No matter, never mind, thanks for the catch!

**Reply by bnewbold** [reply]
Yup, we're using HTTP resolution, and it would be a Big Bad Broken thing if neither were working.

We originally added HTTP resolution to support large hosted platforms, like bsky.social, but also including https://neocities.org/
